test/binlog_streamer_test.go

```diff
@@ -70,6 +83,35 @@ func (this *BinlogStreamerTestSuite) TestConnectErrorsOutIfErrorInServerIdGenera
 	this.Require().Zero(this.binlogStreamer.Config.MyServerId)
 }
 
+func (this *BinlogStreamerTestSuite) TestBinlogStreamerSetsBinlogPositionOnDMLEvent() {
+	err := this.binlogStreamer.ConnectBinlogStreamerToMysql()
+	this.Require().Nil(err)
+
+	eventAsserted := false
+
+	this.binlogStreamer.AddEventListener(func(evs []ghostferry.DMLEvent) error {
+		eventAsserted = true
+		this.Require().Equal(1, len(evs))
+		this.Require().NotEqual("", evs[0].BinlogPosition().Name)
```

Comment:
If you wanna make this more solid `SELECT @@log_bin_basename` should give you the basename that you can then `assertMatch`

Reply:
We actually set `--log-bin=mysql-bin` in the docker-compose config so I'll just match against mysql-bin directly. Thanks for the suggestion.